about/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: layouts/post.njk
+title: About Me
+templateClass: tmpl-post
+eleventyNavigation:
```

Comment:
@cmneal26 @twalker 

Hah, the starter repo sure has a lot of files. But it looks like it's not too cumbersome to make a new content page.

1. Create a folder in the root directory, with the name "animals"
2. Create an index.md file. You can just paste the content from about/index.md, and change the values for "title" and "eleventyNavigation".
3. Write your markdown content.